pilot5-submission/pilot5-programs/adtte.r

```diff
@@ -1,50 +1,24 @@
-# Note to Reviewer
-# To rerun the code below, please refer ADRG appendix.
-# After required package are installed.
-# The path variable needs to be defined by using example code below
-#
-# nolint start
-# path <- list(
-# sdtm = "path/to/esub/tabulations/sdtm", # Modify path to the sdtm location
-# adam = "path/to/esub/analysis/adam"     # Modify path to the adam location
-# )
-# nolint end
-
-###########################################################################
-#' developers : Steven Haesendonckx/Bingjun Wang/Ben Straub
-#' date: 13NOV2022
-#' modification History:
-###########################################################################
-
 # Set up ------------------------------------------------------------------
-
-library(haven)
-library(admiral)
 library(dplyr)
 library(tidyr)
+library(admiral)
 library(metacore)
 library(metatools)
 library(pilot5utils)
-library(xportr)
 
 # read source -------------------------------------------------------------
-
-adsl <- read_xpt(file.path(path$adam, "adsl.xpt"))
-adae <- read_xpt(file.path(path$adam, "adae.xpt"))
-ds <- convert_blanks_to_na(read_xpt(file.path(path$sdtm, "ds.xpt")))
-
+ds <-  convert_blanks_to_na(readRDS(file.path(path$sdtm, "ds.rds")))
+adsl <-  convert_blanks_to_na(readRDS(file.path(path$adam, "adsl.rds")))
+adae <-  convert_blanks_to_na(readRDS(file.path(path$adam, "adae.rds")))
 
 ## placeholder for origin=predecessor, use metatool::build_from_derived()
-
-metacore <- spec_to_metacore(file.path(path$adam, "adam-pilot-3.xlsx"), where_sep_sheet = FALSE)
+metacore <- spec_to_metacore(file.path(path$adam, "adam-pilot-5.xlsx"), where_sep_sheet = FALSE)
 
 # Get the specifications for the dataset we are currently building
-
 adtte_spec <- metacore %>%
   select_dataset("ADTTE")
 
 # First dermatological event (ADAE.AOCC01FL = 'Y' and ADAE.CQ01NAM != '')
-
 event <- event_source(
   dataset_name = "adae",
   filter = AOCC01FL == "Y" & CQ01NAM == "DERMATOLOGIC EVENTS" & SAFFL == "Y",
```

Comment:
I recommend prefixing functions like in #27.

Reply:
I'm a bit hesitant to do prefix of packages.  It gets really verbose and then we have to be consistent.  If there are name conflicts with functions and different packages I can see prefixing being of value. 

WDYT @RConsortium/pilot5 I always go with no prefixing unless namespace conflicts - but this isn't a hill I won't to die on!! :)

Reply:
But a good convention for us to discuss as other TLF programs have them